pricing-page-exercise/dilara/styles.css

```diff
@@ -0,0 +1,234 @@
+.nav-bar {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content:space-between;
+    min-height:38px;
+    padding: 8px 24px;
+    align-items: center;
+    border-bottom: 1px solid #dee2e6;
+    box-shadow: 0 4px 12px rgb(0 0 0 / 5%);
+}
+
+.company-name {
+    font-size: 20px;
+    font-weight: 400;
+}
+
+.link {
+    margin-left: 20px;
+    color: #343a40;
+    text-decoration: none;
+}
+
+.link:hover {
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+.sign-up {
+    margin-left: 20px;
+    background-color: white;
+    color: #007bff;
+    border: 1px solid #007bff;
+    border-radius: 5px;
+    height: 38px;
+    width: 80px;
+    font-size: medium;
+}
+
+.sign-up:hover {
+    background-color: #007bff;
+    color: white;
+}
+
+body {
+display: block;
```

Comment:
I think it should be `block` by default, so no need to set it explicitly like this.